Sparkbox Sprint 50

rkelbel opened the pull request:
# READY FOR REVIEW

## Summary
- #973 
- #971 
- #968 

### TASK LIST:
- [x] Add compiled CSS & JS
- [x] Run BackstopJS visual regression tests
- [x] Ensure Code Climate passes against the `develop` branch. Link here: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/967
- [x] Assign to Mike for review.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. `npm run test` should pass in the root folder
2. If you are interested in the testing steps for associated PRs, please take a look at the individual PR's description and "Steps to Test" section.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

Commit:
fix(STN-848): making raised hb-cards uniform in width as well when a (#968)

browser doesn't support :not selector

Commit:
Feat(STN-562): Splash font animations (#971)

* feat(STN-562): splash font animations

* chore: compile css and js

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr967-gouee5tggwhazf8seybmahzmljk7rylc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61565258cc9cc437505b2ae9

Commit:
Feat(STN-835): spotlight slider styles (#973)

* feat(STN-835): implement Spotlight Slider styles
- refactor: move bottom spacing into gradient hero pattern to avoid
situations where the extra space is not needed.
- some refactoring of code (i.e. removing conditionals) and
enabling restrictHeight() to work on spotlight slider regardless of
window size

* chore: compile js/css

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr967-gouee5tggwhazf8seybmahzmljk7rylc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61565258cc9cc437505b2ae9
